infra/terraform/test-org/ci-triggers/triggers.tf

```diff
@@ -154,3 +154,28 @@ resource "google_cloudbuild_trigger" "terraform_example_foundation_lint" {
   filename = "build/cloudbuild.lint.yaml"
 }
 
+resource "google_folder" "ci-example-foundation" {
+  display_name = "ci-example-foundation"
+  parent       = "organizations/${local.org_id}"
```

Comment:
@morgante I am not sure if this should be under `folders/${ci-projects-id}`

Reply:
@bharathkkb Please move it to the same location as all other tests. In fact, I'm not sure why this couldn't be treated the same as any other repo?

Reply:
@morgante wouldnt it would fail here https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/blob/886d4c610e155a650a58444c0c7c2b90ba5a7cc3/infra/terraform/test-org/ci-triggers/locals.tf#L24
as the folder name will be `ci-example-foundation` and this transform would make it `terraform-google-example-foundation` which is invalid?

Reply:
Good catch. Could we tweak that logic to allow adding repos with different names?

Reply:
We could have something like this https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/blob/00c5a11276d01dc2ba569075871e5b44f2788eea/infra/terraform/test-org/ci-triggers/locals.tf#L18
but considering that we haven't cut any releases for `terraform-example-foundations` wouldn't it be better to rename to `terraform-google-example-foundations` to keep alignment?

Reply:
@morgante I have added an exception for `example-foundations` PTAL